save-frontend/src/main/kotlin/com/saveourtool/save/frontend/routing/BasicRouting.kt

```diff
@@ -92,34 +92,43 @@ val basicRouting: FC<AppProps> = FC { props ->
         }
     }
 
-    val creationView: FC<Props> = withRouter { _, params ->
+    val creationView: VFC = withRouter { _, params ->
         CreationView::class.react {
             organizationName = params["owner"]
         }
     }
 
-    val organizationView: FC<Props> = withRouter { location, params ->
+    val organizationView: VFC = withRouter { location, params ->
         OrganizationView::class.react {
             organizationName = params["owner"]!!
             currentUserInfo = props.userInfo
             this.location = location
         }
     }
 
-    val awesomeBenchmarksView: FC<Props> = withRouter { location, _ ->
+    val awesomeBenchmarksView: VFC = withRouter { location, _ ->
         AwesomeBenchmarksView::class.react {
             this.location = location
         }
     }
 
-    val contestGlobalRatingView: FC<Props> = withRouter { location, _ ->
+    val contestGlobalRatingView: VFC = withRouter { location, _ ->
         ContestGlobalRatingView::class.react {
             organizationName = URLSearchParams(location.search).get("organizationName")
             projectName = URLSearchParams(location.search).get("projectName")
             this.location = location
         }
     }
 
+    val demoView: FC<Props> = withRouter { _, params ->
```

Comment:
jFYI: I failed to migrate `withRouter` method to the new version of kotlin wrapper.

Reply:
oh sugar, it's gonna be a problem

Seems that we will need to rework existing `withRouter`